controllers/cloud.redhat.com/clowdapp_controller.go

```diff
@@ -178,6 +179,12 @@ func (r *ClowdAppReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	}
 
 	SetDeploymentStatus(ctx, &proxyClient, &app)
+
+	app.Status.Ready = false
+	if app.Status.Deployments.ManagedDeployments == app.Status.Deployments.ReadyDeployments {
```

Comment:
This "functionality" was recently moved into a function on the type for the env, care to just quickly do the same here, 5min job